exercises/object-oriented-programming/src/main/java/com/cbfacademy/cars/Car.java

```diff
@@ -11,27 +11,29 @@ public Car(String make, String model, int year){
         this.model = model;
         this.year = year;
     }
-
-    public String toString(){
-        String newCar = make + "\n"+model + "\n" + year;
-        return newCar;
-    }
+    
 
     public void getMake(){
-        System.out.println("Have you seen my nice new " +this.make +"?");
+        return;
     }
 
     public void getModel(){
-        System.out.println("It's a " +this.model+ " You know");
+    return;
     }
 
     public void getYear(){
-        System.out.println("It was released in " + this.year + ".");
+    return;
     }
 
     public void getDetals(){
-        getMake();
-        getModel();
-        getYear();
+        System.out.print("Have you seen my nice new " +this.make+"? ");
+        System.out.print("It's a " +this.model+ " You know ");
+        System.out.print("It was released in " + this.year + ". ");
```

Comment:
Your getters still aren't returning any values. Any method prefixed with "get" would be expected to return some value — `getModel()` should return the value of `model`, `getDetails` should return the string that you're currently outputting, etc